components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -1234,11 +1234,13 @@ def enrolled_with(enrollment)
         evidence = self.send(evidence_type)
         next unless evidence.present?
         enrollment_member = enrollment.hbx_enrollment_members.where(:applicant_id => family_member_id).first
-        aptc_or_csr_used = enrollment_member.applied_aptc_amount > 0 || ['csr_73', 'csr_87', 'csr_94', 'csr_limited'].include?(csr_eligibility_kind)
+        aptc_or_csr_used = enrollment_member.applied_aptc_amount > 0 || ['03', '04', '05', '06'].include?(enrollment.product.csr_variant_id)
```

Comment:
@j1joey @jacobkagon  Why are we not using the applicant's CSR determination and using the Enrollment Product's csr variant?

Reply:
@saikumar9 we use the enrollment csr because when a person is granted aptc/csr but decides to choose an enrollment that doesn't use either, the current logic would still treat the enrollment as if it had aptc/csr applied, causing errors (evidences going to outstanding). By choosing the csr from the enrollment, we can see what csr was actually supposed to be applied to the logic, and update evidences open correctly